src/components/send-money/SendMoneyAmountInput.js

```diff
@@ -62,9 +67,9 @@ class SendMoneyAmountInput extends Component {
    }
 
    handleChangeAmount = (e, { name, value }) => {
-      const amountStatus = !Number.isInteger(Number(value))
-         ? 'Please enter a whole number.'
-         : value > Number(this.props.amount)
+      const amountStatus = value && !/^[0-9]*(|[.][0-9]{1,5})$/.test(value)
```

Comment:
I think it makes sense to extract this regexp into separate function, something like `isDecimalString`

Reply:
changed